bucket/chatterino.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.1.5-4",
+    "homepage": "https://chatterino.com",
+    "description": "Chat client for twitch chat",
```

Comment:
```suggestion
    "description": "Twitch chat client",
```

Reply:
Added change to a single commit including other requested change. (Moving suggest before architecture)